packages/drivers/odsp-driver/src/odspError.ts

```diff
@@ -23,7 +23,10 @@ export function errorObjectFromSocketError(socketError: IOdspSocketError, handle
         const error = createOdspNetworkError(
             message,
             socketError.code,
-            socketError.retryAfter);
+            socketError.retryAfter,
+            undefined,
+            socketError.error ? JSON.stringify({ error: socketError.error }) : undefined,
```

Comment:
If we always intent to have a string here, why it's any in the signature and object in tests?

Reply:
The error that we receive from socket does not have this as string. It is an object.

Reply:
Got it. Please add comments like // reponseText, as it's very hard to follow what each arg means.
One other thing to add - I run in the past into the issue that error object had circles and was not JSON'able. See getCircularReplacer() and its usage - we probably want to use it here, and likely in other places like that.